recipes/mosquitto/all/conandata.yml

```diff
@@ -1,5 +1,5 @@
 sources:
-  "1.6.10":
-    sha256: ffcfca01342ca8b82303bd1a9ccb0ff9b715e5711280a2fedba14e739f54f595
```

Comment:
You can't remove old version already available for download.

Reply:
I am not sure I understand since I did (2 months ago). I see removing `1.6.10` and replacing with `1.6.12`

Reply:
This is against and ADD PR in CCI so this version has never been checked in